lib/widgets/home.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:mixeditems/providers/provider.dart';
+import 'package:provider/provider.dart';
+
+class Home extends StatelessWidget {
```

Comment:
Dart has a coding guideline to have a keyed constructor in any widget: https://dart-lang.github.io/linter/lints/use_key_in_widget_constructors.html

Reply:
I have changed it! :)